2018-11-26 19:12:03,104 INFO 84432[guizhou_start.py:18 start_cmd]: --------------------------------------------------------------------------------------------
2018-11-26 19:12:03,104 INFO 84432[guizhou_start.py:19 start_cmd]: now start the host : 10.11.5.74
2018-11-26 19:12:03,115 INFO 84432[transport.py:1746 _log]: Connected (version 2.0, client OpenSSH_7.2p2)
2018-11-26 19:12:03,204 INFO 84432[transport.py:1746 _log]: Authentication (publickey) failed.
2018-11-26 19:12:03,229 INFO 84432[transport.py:1746 _log]: Authentication (password) successful!
2018-11-26 19:12:03,585 INFO 84432[guizhou_start.py:25 start_cmd]: cmd is : sudo /data/CI/kms_33_13320/kms.sh start
2018-11-26 19:12:03,586 INFO 84432[guizhou_start.py:26 start_cmd]: error list start is :[]
2018-11-26 19:12:03,586 INFO 84432[guizhou_start.py:27 start_cmd]: stdout list start is :[u'/data/CI/kms_33_13320\n', u'kms.jar\n']
2018-11-26 19:12:03,587 INFO 84432[guizhou_start.py:28 start_cmd]: process is starting,wait a momunt
[Finish] Success to deploy [559]